cocotb/cocotb.bzl

```diff
@@ -132,10 +132,12 @@ def _get_test_command(ctx, verilog_files, vhdl_files):
     seed_args = " --seed {}".format(ctx.attr.seed) if ctx.attr.seed != "" else ""
 
     test_module_args = _pymodules_to_argstring(ctx.files.test_module, "test_module")
+    python_interpreter = ctx.toolchains["@bazel_tools//tools/python:toolchain_type"].py3_runtime.interpreter.path
```

Comment:
shouldn't we be using:
```
toolchains=["@rules_python//python:current_py_toolchain"]
```
and
```
${PYTHON3}
```
as recommended in https://rules-python.readthedocs.io/en/latest/getting-started.html#toolchain-usage-in-other-rules ?